src/diff.jl

```diff
@@ -113,6 +113,25 @@ function expand_derivatives(O::Symbolic, simplify=false; occurances=nothing)
                     return expand_derivatives(operation(arg)(inner), simplify)
                 end
             end
+        elseif isa(operation(arg) , Integral)
+            if isa(operation(arg).domain , AbstractInterval)
+                domain = operation(arg).domain
+                a , b = DomainSets.endpoints(domain)
+                c = 0
+                inner_function = expand_derivatives_(arguments(arg)[1])
+                if isa(value(a), Term)
+                    t1 = replaceSym(operation(arg).x ,value(a) , inner_function )
+                    t2 = D(a)
+                    c -= t1*t2
+                elseif isa(value(b) , Term)
```

Comment:
I think you mean `if` instead of `elseif`? So if both `a` and `b` are terms, it would work out...

Reply:
I resolved this